api/src/pipes/zipfile-validation.pipe.ts

```diff
@@ -0,0 +1,9 @@
+import { ArgumentMetadata, PipeTransform } from '@nestjs/common';
+
+export class ZipfileValidationPipe implements PipeTransform {
+  transform(value: Express.Multer.File, metadata: ArgumentMetadata): any {
+    console.log('value', value);
+    console.log('metadata', metadata);
```

Comment:
I think these `console.log` are not necessary. If they are, please use `Logger#debug`

Reply:
@Dyostiq I'm not done with the API side validation part so this is WIP
Actually I think I'll need your help here :)